include/ublk_cmd.h

```diff
@@ -360,6 +360,8 @@ struct ublksrv_ctrl_dev_info {
  */
 #define		UBLK_IO_OP_REPORT_ZONES		18
 #define		UBLK_IO_OP_EVENTFD		19
+#define		UBLK_IO_OP_EPOLLFD		20
```

Comment:
Please define it in include/ublksrv.h, and ublk_cmd.h should includes kernel UAPI stuff.  Also maybe it is more flexible to define builtin OP in the range of 0xe0 ~ 0xef.